app/views/list.html

```diff
@@ -18,6 +18,13 @@ <h2 class="pull-right text-right">
     <i class="fa fa-spinner fa-spin" ng-if="pendingTests"></i>
     Lancer tous les tests
   </button>
+  <a href="../../">
```

Comment:
Don't use a relative URL here, link to `/`, otherwise this will break if we change the URL scheme.